osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -80,6 +80,8 @@ public override Bindable<double> Bindable
 
                 set
                 {
+                    KeyboardStep = 0.01f;
+

```

Comment:
Instead of this, set KeyboardStep explicitly at construction time:

```
new SensitivitySetting
{
    KeyboardStep = 0.01f
}
```

This is the way it's done in other places.